doc/source/user_guide/io.rst

```diff
@@ -3376,93 +3375,6 @@ The default is to 'infer':
    os.remove("data.pkl.gz")
    os.remove("s1.pkl.bz2")
 
-.. _io.msgpack:
-
-msgpack
--------
```

Comment:
Given that this is deprecated/removed on a very short time frame, I think it might be good to keep this title a bit longer with a small note that it was deprecated/removed and how to replace it.

Reply:
why? the whole point is to clean things for 1.0

Reply:
To provide documentation about this removal, and how to replace it.

For example, there have been issues opened with questions about this which lead to added examples in the docstrings, but this has never been published yet in actual documentation.

Reply:
k that’s fair

Reply:
restored this section.  LMK if you have suggestions for additional notes to put in here.

any thoughts on the LICENSE files mentioned in the OP?

Reply:
I wouldn't restore the full section, but rather only keep a short explanation that it was deprecated/removed, and how to replace it (see the docstrings for some content for that)